tensorrt_llm/_torch/memory_buffers_util.py

```
@@ -0,0 +1,63 @@
+class Buffers:
+
+    def __init__(self):
+        buffers: {str, list[(torch.Tensor, bool)]} = {}
+
+    def get_buffer(self, tensor_shape: list[int], dtype: torch.dtype,
```

Ruff (0.12.2)

Unused method argument: `tensor_shape`

(ARG002)